PhysicsTools/MVAComputer/src/ProcMLP.cc

```diff
@@ -100,14 +101,15 @@ namespace {  // anonymous
   }
 
   void ProcMLP::eval(ValueIterator iter, unsigned int n) const {
-    double *tmp = (double *)alloca(2 * maxTmp * sizeof(double));
+    double *tmp = (double *)calloc(2 * maxTmp, sizeof(double));
```

Comment:
differences between `alloca` and `calloc`. 
- `alloca` does not initialize allocalted memory but it automatically deletes it when veriable goes out of scope
- `calloc` initialize memory with `0` but does not delete the allocated memory (so one need to explicitly delete it)